### DISCUSSION TITLE
remote repository "Acess restricted to this site"

### DISCUSSION THREAD

**Discussion by spyingcreeper09** [community/community#132447] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Ok, so i have a mc mod i want to start and i want to use a remote repository (im following this tutorial here: https://www.youtube.com/watch?v=0Pr_iHlVKsI&list=PLKGarocXCE1EO43Dlf5JGh7Yk-kRAXUEJ) but when i go to share it with myself it gives me an error saying that it cant visit site because access has been restriced. 
![image](https://github.com/user-attachments/assets/6adefd0c-bb2d-4e29-8d14-8352167d51b9)

**Comment by yosifsamir** [comment]
- create repository manually . 
 - then take all commands and add to your terminal one by one . 
 - then commit and push . your work will push correctly .

**Comment by spyingcreeper09** [comment, marked as answer]
Sorry for leaving this open but uninstalling and reinstalling the ide worked for me
